parse_cli.l

```diff
@@ -494,7 +494,8 @@ gp	{ if (!cmd) {
 					yyless(0);
 					}
 
-<getup2tc>[0-7]			{ up2tc_a[up2tc_idx++] = atoi(yytext);
+<getup2tc>[0-7]			{ up2tc_a[up2tc_idx] = atoi(yytext);
+				up2tc_idx++;
```

Comment:
What is the difference between these two lines?  Coverity must have a wrong model.